presto-main/src/test/java/com/facebook/presto/sql/TestExpressionInterpreter.java

```diff
@@ -139,6 +142,8 @@ public void testOr()
         assertOptimizedEquals("false or bound_string='z'", "bound_string='z'");
 
         assertOptimizedEquals("bound_string='z' or bound_long=1+1", "bound_string='z' or bound_long=2");
+
+        assertOptimizedEquals("bound_varbinary =x'0 0' or bound_long=1+1", "bound_string='z' or bound_long=2");
```

Comment:
Don't add these tests. Tests in this class is for a different purpose which does not depend on covering various types.